### PR TITLE
Restructuring and additional pages

### PR DESCRIPTION
**Pull request by lang-m**

### CODE REVIEW

**Comment by lang-m** [comment]
@samjrholt Could you please add instructions for Windows to [source/getting-started/jupyter-basics.rst](https://github.com/ubermag/ubermag.github.io/pull/13/files#diff-c677639f8c875862f5809ada38a7b153e41706c4cd1a7afd80516dbdc93ae0ad)

File: source/installation.rst [hunk 1]
```diff
@@ -105,6 +105,60 @@ install.
...
+-----------------------
+
+Ubermag can use different existing micromagnetic calculators (currently OOMMF
+and mumax\ :sup:`3`).
+
+.. tab-set::
+
+   .. tab-item:: OOMMF
+
+      When you install `ubermag` using `conda` OOMMF will be automatically
```
Comment (fangohr) [inline source/installation.rst:118]: We use single back ticks here (`ubermag`) but you double back ticks (``ubermag``) in other rst files. Is that a problem?

**Review by fangohr — APPROVED** [review]
Thanks - looks great to me. Please check you are happy with my suggested changes.

File: source/installation.rst (hunk 1 above)
Comment (lang-m) [reply source/installation.rst:118]: It should be double back ticks. Thanks for pointing that out (too much Markdown in the last time, I guess).

**Pull request merged**